packages/node/src/integrations/context.ts

```diff
@@ -18,6 +19,12 @@ import type {
 export const readFileAsync = promisify(readFile);
 export const readDirAsync = promisify(readdir);
 
+// Process enhanced with methods from Node 18, 20, 22 as @types/node
+// is on `14.18.0` to match minimum version requirements of the SDK
+interface ProcessWithCurrentValues extends NodeJS.Process {
+  availableMemory(): number;
```

Comment:
Maybe make this method optional to signify it doesn't exist for all supported node versions?

Reply:
yeah good call! will change